backend/app/__init__.py

```diff
@@ -30,9 +25,9 @@ def create_app(config_name):
     app.config.from_object(config[config_name])
     app.config["JWT_TOKEN_LOCATION"] = ["cookies"]
     app.config["JWT_COOKIE_SECURE"] = True if config_name == "production" else False
+    app.config["JWT_COOKIE_SAMESITE"] = "None" if config_name == "production" else None
```

Comment:
out of curiosity, did this line fix the cookie storage problem?

Reply:
No it was because of a mismatch in the host names—the backend was running on `127.0.0.1` and the frontend was running on `localhost`. Once both were on `localhost` it worked. This is gonna be needed once it's deployed though since the frontend and backend will be on different domains (I think) so the Samesite attribute needs to be "None" in that case for the cookies to be set